DigitalOffice/Services/CheckRightsService/Startup.cs

```diff
@@ -26,6 +34,10 @@ public void ConfigureServices(IServiceCollection services)
             {
                 options.UseSqlServer(Configuration.GetConnectionString("SQLConnectionString"));
             });
+
+            services.AddTransient<ICheckRightsRepository, CheckRightsRepository>();
+            services.AddTransient<IGetRightsListCommand, GetRightsListCommand>();
+            services.AddTransient<IMapper<DbRight, Right>, RightsMapper>();
```

Comment:
Maybe will be a good idea to create separate method for binding DI container?

Reply:
Yep, great idea

Reply:
Done

Reply:
Where? I do not see any new methods? Create separated methods for each type of object e.g. ConfigureCommands, ConfigureRepositories and others ...